integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java

```diff
@@ -111,18 +110,18 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     // install and verify operator
     installAndVerifyOperator(opNamespace, domainNamespace);
 
-    // create secret for admin credentials
+    // create secret for admin credential with special characters
     logger.info("Create secret for admin credentials");
     String adminSecretName = "weblogic-credentials";
     assertDoesNotThrow(() -> createDomainSecret(adminSecretName, 
-            ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, domainNamespace),
+            ADMIN_USERNAME_DEFAULT, "W##%*}!`''1~3x", domainNamespace),
             String.format("createSecret failed for %s", adminSecretName));
 
-    // create encryption secret
+    // create encryption secret with special characters
     logger.info("Create encryption secret");
     String encryptionSecretName = "encryptionsecret";
     assertDoesNotThrow(() -> createDomainSecret(encryptionSecretName, "weblogicenc",
-            "weblogicenc", domainNamespace),
+            "W##%*}!`''1~3x", domainNamespace),
```

Comment:
It'd be good to test similar for internal situational config (DII pass), custom situational config maps (such as for a database password), and custom model config maps (such as for database password).  In addition, it'd be good to test the use in environment variables (sit-cfg and model files can reference env vars instead of secrets).

Reply:
Finally, it'd be good to test for something that takes an arbitrary unencrypted string - maybe the notes field of some mbean would be fine...  (For sit-cfg, model YAML, and model-config-map YAML).

Reply:
situational config Feature is overwritten by Dynamic Mii Upgrade.  Let me try custom model config maps (such as for database password) and ENV variable in domain resources. Here I do not want to create a whole new domain/class  to verify all edge (nice-to-have) cases. Let me see if I can insert these usecases in existing test class.

Reply:
Sit-cfg still applies to the other domain home source types (DII, and  DIPV), and so should also be tested.

Reply:
Also, please test with a single quote instead of two quotes right next to each-other.  Two consecutive quotes could parse correctly even if there's a bug in macro parsing.  

In addition, please put the comment '#' symbol first - I think that'd be more likely to be parsed incorrectly if there's a bug.

Reply:
In addition, please add something like `${DOMAIN_UID}` and `$(ls)` to the sit-cfg/model X secrets/env-vars/props - as this catches more cases.  Plus the set values should be checked using mbeans that the values are as expected -- if you don't check via mbeans, it's possible that the set values will be parsed using shell on _both_ the configuration and the usage paths in WKO and seem to work fine (which would be bad).

Reply:
Now the password is changed to  [ ##W%*}!"'"`']\\\\//1$$~x ] Added test for config-map with db password with special characters

Reply:
Updated the test to use "${DOMAIN_UID} $(ls)" in encryption secret. Add a test to define a env variable with special characters in its value.

Reply:
Added test for non standard admin user such as 'wlsadmin'.  
Add sitconfig test with supported environment variables  ( https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/configoverrides/#override-template-macros)
We have already sitconfig tests with secret macro.